### PR TITLE
Replace Sample App

### PR DESCRIPTION
**Pull request by Xanonymous-GitHub**
## Description
- Replace the existing smaple app.
  - The new version of the sample app is completely written in kotlin and uses Jetpack Compose to design the UI.

- In addition, some SDK's dependency versions were also upgraded.
- Update Java version to 11 in build CI.

## How to verify?
- Run these two tasks to check if they build successfully: 
  - `:app:assembleDebug`
  - `:line-sdk:assembleDebug`

- Run the new Sample App to an emulator/physical device.

## Screenshot

<table>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-    -->
-    <uses-sdk tools:overrideLibrary="android.support.customtabs" />
+    xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name=".LineSdkTestApplication"
-        android:allowBackup="false"
+        android:dataExtractionRules="@xml/data_extraction_rules"
```
Comment (plateaukao) [inline app/src/main/AndroidManifest.xml:8]: these two lines and corresponding files could be removed,

Comment (Xanonymous-GitHub) [reply app/src/main/AndroidManifest.xml]: Seems generated by Android Studio.
Addressed in [40825ee](https://github.com/line/line-sdk-android/pull/128/commits/40825ee5d14ce486576d9c37da07d6cf9f5a25f7).

**Review by plateaukao — APPROVED** [review]
LGTM

**Review by plateaukao — APPROVED** [review]
LGTM

**Pull request merged**